scripts/openmc-ace-to-hdf5

```diff
@@ -111,25 +118,25 @@ for filename in ace_libraries:
             # Continuous-energy neutron data
```

Comment:
@paulromano So I found an issue related more to runtime/workload than correctness.
As you know, ENDF-VII.0 which ships with MCNP has files endf70a up to endf70k and endf70sab.  endf70a, for example, contains data for many isotopes (up to V-Nat) with 5 temperatures for each.  the xsdir or cross_sections.xml file will have a separate entry for every one of those iso/temp combinations, all with the filename of endf70a.
The way this is written the script will see the 250-ish something tables within endf70a, and process each and everyone of them 250-ish times.